src/languages/en.js

```diff
@@ -160,7 +160,7 @@ export default {
         newMsg: ({count}) => `${count} new message${count > 1 ? 's' : ''}`,
     },
     reportTypingIndicator: {
-        isTyping: 'is typing...',
+        isTyping: 'is typing',
```

Comment:
I think we should keep this as it is??

Reply:
I did add `...` back to `typing...`.

![image](https://user-images.githubusercontent.com/57435789/137977073-99765983-0472-4a29-85c1-b75d3e23ca37.png)

I am not sure if two `...` look fine. But upto you guys.